app/components/file-diff-card.hbs

```diff
@@ -5,6 +5,6 @@
 
   <div class="text-sm leading-6 dark:text-gray-400">
     {{! @glint-expect-error: not ts-ified yet }}
-    <SyntaxHighlightedDiff @code={{@code}} @language={{@language}} @shouldCollapseUnchangedLines={{true}} />
+    <SyntaxHighlightedDiff @code={{@code}} @language={{@language}} @shouldCollapseUnchangedLines={{true}} @theme="github-light" />
```

Comment:
We shouldn't have to specific theme manually, these components are supposed to automatically response to light vs. dark mode (check an autofix request for an example)